### PR TITLE
Fix missing Realex::Notification acknowledge method

### PR DESCRIPTION
**Pull request by davidjrice**

### CODE REVIEW

File: lib/offsite_payments/integrations/realex.rb [hunk 1]
```diff
@@ -3,7 +3,8 @@ module Integrations #:nodoc:
     module Realex
       mattr_accessor :production_url
       mattr_accessor :test_url
-      self.production_url = 'https://hpp.realexpayments.com/pay'
+      # self.production_url = 'https://hpp.realexpayments.com/pay'
```
Comment (girasquid) [inline lib/offsite_payments/integrations/realex.rb]: Can you remove this entirely? Why leave it in?

Comment (davidjrice) [inline lib/offsite_payments/integrations/realex.rb]: I believe Realex only want this in as a temporary change. To allow them to put through test transactions.

Comment (andrewpaliga) [inline lib/offsite_payments/integrations/realex.rb]: @davidjrice looks like the new url will be in place for a while

Comment (seanmacdomhnall) [inline lib/offsite_payments/integrations/realex.rb]: Okay so lets just leave it with https://epage.payandshop.com/epage.cgi as the sole URL for the moment.

**Pull request closed without merging**